README.md

```diff
@@ -45,18 +46,19 @@ Run `npm info "@upstatement/eslint-config@latest" peerDependencies` to list the
     - **Option 2:** Without `npx`
 
       ```sh
-      npm install --save-dev @upstatement/eslint-config eslint babel-eslint prettier eslint-config-prettier
+      npm install --save-dev @upstatement/eslint-config @babel/core@7.x.x @babel/eslint-parser@7.x.x eslint@7.x.x eslint-config-prettier@8.x.x prettier@2.x.x
 
       # or
 
-      yarn add --dev @upstatement/eslint-config eslint babel-eslint prettier eslint-config-prettier
+      yarn add --dev @upstatement/eslint-config @babel/core@7.x.x @babel/eslint-parser@7.x.x eslint@7.x.x eslint-config-prettier@8.x.x prettier@2.x.x
       ```
 
-1. Create an `.eslintrc` file at the root of your project with the following:
+2. Create an `.eslintrc` file at the root of your project with the following:
 
     ```json
     {
-      "extends": "@upstatement"
+      "root": true,
```

Comment:
We need `"root": true` in our `.eslintrc` files now do avoid a deprecation warning

https://eslint.org/docs/user-guide/migrating-to-7.0.0#runtime-deprecation-on-personal-config-files